frontend/fingertips-frontend/lib/ViewsHelpers.ts

```diff
@@ -244,3 +255,9 @@ export async function getIndicatorData(
 
   return indicatorDataAllAreas;
 }
+
+export function determineBenchmarkRefType(lineChartAreaSelected?: string) {
+  if (lineChartAreaSelected && lineChartAreaSelected !== areaCodeForEngland)
+    return 'AreaGroup';
+  return 'England';
```

Comment:
It's personal preference (so you can ignore me if you disagree), but I'd prefer to always use braces for if statements.

```suggestion
  if (lineChartAreaSelected && lineChartAreaSelected !== areaCodeForEngland) {
    return 'AreaGroup';
  }

  return 'England';
```